packages/eos-components/src/components/ChannelCardGroup.vue

```diff
@@ -16,8 +16,8 @@
 
       <!-- eslint-disable vue/no-use-v-if-with-v-for -->
       <b-card
-        v-if="index === rows.length - 1"
         v-for="n in (columns - row.length)"
+        v-if="index === rows.length - 1"
```

Comment:
I have fixed this and the "node" warning below in my maintenance PR #74, please merge that one first, or I will have to fix conflicts.